designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/ScenarioStatistics.scala

```diff
@@ -174,14 +214,19 @@ object ScenarioStatistics {
     else orderedList.last
   }
 
+  private def mapNameToStat(name: String): String = {
```

Comment:
Component name could be not enough to identify component. E.g. we have kafka source and kafka sink. `ComponentId` have both things